core-feature-pack/common/src/main/resources/content/bin/common.ps1

```diff
@@ -254,7 +254,9 @@ Param(
   	$PROG_ARGS += "-Dlogging.configuration=file:$logFileProperties"
   }
   $PROG_ARGS += "-Djboss.home.dir=$JBOSS_HOME"
-  $PROG_ARGS += "-Djboss.server.base.dir=$global:JBOSS_BASE_DIR"
+  if (-not($SERVER_OPTS -match "-Djboss.server.base.dir")) {
+    $PROG_ARGS += "-Djboss.server.base.dir=$global:JBOSS_BASE_DIR"
+  }
```

Comment:
That's ok, maybe the alternative would have been to just completely drop this line, since I think  JBOSS_BASE_DIR will always contain the default value. Just FYI, as a finding doing the review.

Reply:
Ok, @lvydra , based on your comment about this, I resolve this thread then. Thanks for investigating whether there is any possibility by using the JDK API